oaknational/Oak-Web-Application
# chore: dl-card component and story LESQ-8

## Description

- Creates DLCard slim download component in storybook
- Add grey60 color
- Updates Checkbox/Visual Checkbox components to 

## AC:

- [x]  Uses a new slimline card design
- [x]  Shows the format of the file
- [x]  Has clickable tick box
- [x]  Has different states (see Figma)

## Issue(s)

Fixes #LESQ-8

## How to test

1. Go to https://deploy-preview-2045--owa-storybook.netlify.thenational.academy/?path=/docs/download-components-dl-card--docs
2. Click on DLCard component on 
3. You should see slimline DL card
4. Toggle error to see error state
5. Click to see focus state

## Screenshots

How it should now look:

![Screenshot 2023-10-25 at 09 28 32](https://github.com/oaknational/Oak-Web-Application/assets/91190841/7dab5216-bdba-4051-9940-293b99fd444a)
![Screenshot 2023-10-25 at 09 28 55](https://github.com/oaknational/Oak-Web-Application/assets/91190841/4ece130e-82b6-4c67-9c00-0b9232bcce2c)

## Checklist
